packages/garbo/src/tasks/dailyItems.ts

```diff
@@ -429,6 +442,15 @@ const DailyItemTasks: GarboTask[] = [
     do: () => BurningLeaves.burnSpecialLeaves($item`day shortener`),
     spendsTurn: false,
   },
+  {
+    name: "Burning Leaves Best Unlimited Tradeable Item",
+    completed: () =>
+      !BurningLeaves.have() ||
+      BurningLeaves.numberOfLeaves() <
+        500 + (BurningLeaves.burnFor.get(bestLeafBurn()) ?? Infinity), // Keep a reserve of 500 leaves for limited items and fights
```

Comment:
If we do this I think I'd like to see a higher reserve, like enough to get items and fights for several days / a week at least